pacman/makepkg_clang64.conf

```diff
@@ -59,9 +59,9 @@ DXSDK_DIR=${MINGW_PREFIX}/${MINGW_CHOST}
 CPPFLAGS="-D__USE_MINGW_ANSI_STDIO=1"
 CFLAGS="-march=x86-64 -mtune=generic -O2 -pipe"
 CXXFLAGS="-march=x86-64 -mtune=generic -O2 -pipe"
-LDFLAGS="-pipe"
-# Uncomment to enable hardening (ASLR, High entropy ASLR, DEP)
-#LDFLAGS="-pipe -Wl,--dynamicbase,--high-entropy-va,--nxcompat"
+LDFLAGS="-pipe -Wl,--dynamicbase,--high-entropy-va,--nxcompat,--default-image-base-high"
```

Comment:
Like above `--high-entropy-va` and `--nxcompat` are silently ignored since they are always enabled by LLD.

Upstream LLD doesn't understand `--default-image-base-high`, I have patched our LLD to silently ignore this option since it does the right thing by the default. I don't want to keep that patch for longer than necessary so IMO this specific flag is actually harmful with no benefits.

Reply:
ok, sounds good